packages/design-system/adr/process-ds-transformation.md

```diff
@@ -0,0 +1,87 @@
+# Transform Design system process
+
+## Current status of DS?
+
+- A dedicated team → 2 DS dev + 1 DS designer
+- In _collaboration_ with products designers/content designers
+- _Lack of collaboration_ from product developers
+- _Low usage_ in products
+- How can we move from the DS team to something?
+
+## Org proposals
+
+- Have a composite team from the different domains (1 or 2 devs per domain) and Designers => `Design System Champions 🎖`
+
+  - Goal : Bring the product teams inside the process of build the DS
+
+  - Existing teams (nb of frontend dev):
+
+    - API : Tester / Designer / Creator (1,5?)
+    - DGD : Data Preparation / Data Stewardship / Data Inventory (4,5)
+    - DPD : Pipeline Designer (3)
+    - TDOPS : TMC / Portal / IDAAS (7)
+    - STITCH ?
+
+  - Timeslot Placeholder DS in domains capacity _(see TBD section)_
+    - Jira:
+      - "Main DS ticket" would stay on `TUx`
+      - Tickets would be created in the JIRA's products that take them and be linked to the TUx
+
+## How to construct a roadmap
+
+Discuss the roadmap:
+
+- Activity with priority 1
+
+  - Create new components
+  - Update existing components in DS (not those in TUI)
+
+- Activity with priority P2 (background tasks, less priority)
+  - Integrate & migrate existing components (TUI) to the DS
```

Comment:
Unsure if really P2. 

For the arch team, having a single library as a source of component seemed to be P1 a while ago. @jmfrancois still the case?

Also implementing the design tokens in project-side CSS code should be P1. It's your guarantee that when the next rebranding comes, it's as painless as possible.